### PR TITLE
Introduce page.emulate convenience wrapper

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1421,6 +1421,11 @@ describe('Page', function() {
       await page.setViewport({width: 100, height: 100});
       expect(await page.evaluate(() => screen.orientation.type)).toBe('portrait-primary');
     }));
+    it('should support emulate shorthand', SX(async function() {
+      await page.emulate({viewport:{width: 123, height: 456}, userAgent:'UA'});
```
Comment (aslushnikov) [inline test/test.js]: we have a device here for convenience:

```js
await page.emulate(iPhone);
expect(await page.evaluate(() => window.innerWidth)).toBe(375);
expect(await page.evaluate(() => navigator.userAgent)).toContain('Safari');
```

**Review by aslushnikov — APPROVED** [review]
thanks!

FYI: linter is unhappy

File: examples/pdf.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ await page.navigate('https://news.ycombinator.com', {waitUntil: 'networkidle'});
 // @see https://bugs.chromium.org/p/chromium/issues/detail?id=753118
 await page.pdf({
   path: 'hn.pdf',
-  format: 'A4'
```
Comment (JoelEinbinder) [inline examples/pdf.js:28]: Why this change?

Comment (aslushnikov) [reply examples/pdf.js:28]: I guess the "letter" page format is more widely known then "A4"
